test-interface/src/main/scala/scala/scalajs/testinterface/TestUtils.scala

```diff
@@ -0,0 +1,31 @@
+package scala.scalajs
+package testinterface
```

Comment:
This was originally in the package `org.scalajs.testinterface`, not `scala.scalajs.testinterface`.